modules/install/pages/install.adoc

```diff
@@ -1,71 +1,538 @@
+* The yum command downloads all the RPMs to the host OS, and automatically installs all the files needed to start and manage JBoss EAP.
+* The RPM installation method is only valid for Red Hat Enterprise Linux (RHEL) operating system and not for Windows server.
+* This method is preferred when you want JBoss EAP service files to be installed for the RHEL operating system.
 
 == Installing JBoss EAP using the JAR installer
 
-Brief overview only. Details in lab section below....
+* This method can also be called the GUI installer method.
+* In this method, the jar file is a Java based application that is responsible for providing a step-by-step way to customize the environment setup.
+* This method is preferred when:
+** You want to install on *multiple platforms* like Windows server, Linux etc.
+** You need either graphical interface or a text based console.
+** You want to customise JBoss EAP installation.
+* Due to its flexible nature, the installer can install JBoss EAP by using a response file, with all the options passed as an XML file.
 
-== Installing JBoss EAP using Installation Manager
+== Pre-requisites
 
-Brief overview only. Details in lab section below....
+* You have access to the internet.
+* You have reviewed https://access.redhat.com/articles/6961381[JBoss EAP 8.0 supported configurations].
+* You have created an account on the https://access.redhat.com/[Red Hat Customer Portal] to download JBoss EAP files.
+* You are operating a RHEL 9 server equipped with a graphical user interface (GUI) package.
+* You have installed https://access.redhat.com/articles/6961381[Red Hat supported OpenJDK 17] on RHEL 9 server.
+* Recommended to create a separate *jboss* user account with no shell access (/sbin/nologin).
+* Recommended to download all installation file under /opt partition.
+* Recommended to ensure there is enough space under /opt partition (minimum 800MB).
 
-== Installing JBoss EAP using other methods....
+== (Optional) Pre-configured RHEL9 settings using Vagrant
 
-Brief overview only. Details in lab section below....
+* You can refer https://github.com/RedHatQuickCourses/eap-qc-apps[this eap8 vagrant settings] to configure the RHEL 9 server to get started.
 
-== Starting JBoss EAP
+== Lab 1: Installing JBoss EAP using the Zip file
 
-Provide a brief explanation of different flags for the _standalone.sh_ script, especially *-b* flag....default is bound only to localhost (127.0.0.1) for security.
+*Outcome*: In this lab, you download `jboss-eap-8.0.0-installation-manager.zip` file and install JBoss EAP as a standalone server that can be access at `http://localhost:8080`.
 
-NOTE: Mention that the installation can be automated using a response file (kinda like kickstart), and that this will be covered in the second course in the path (_Advanced JBoss EAP Administration_)
+* Take GUI access of RHEL 9 server.
```

Comment:
Re-word this sentence. What does take mean in this context? Is graphical login really required? Can we get away with Log in to the RHEL 9 VM using SSH?